pkg/linter/linter.go

```diff
@@ -451,6 +463,7 @@ func (l Linter) DetermineEnabledRules(ctx context.Context) ([]string, error) {
 
 	queryStr := `[rule |
         data.regal.rules[cat][rule]
+		count([true | data.regal.rules[cat][rule].notices]) == 0
```

Comment:
I think this is meant to check that there are no notices. But what it seems to do is check that if the notices set rule is defined.

If there is a notices rule, but the set is empty, we still want to enable the rule right?

Reply:
Might 		`object.get(data.regal.rules[cat][rule], "notices", set()) == set()` do the trick?

Reply:
👍

Reply:
Yeah that was a late night addition 😄